src/backend/base/langflow/components/prototypes/CreateData.py

```diff
@@ -23,7 +23,18 @@ class CreateDataComponent(Component):
             value=0,
             range_spec=RangeSpec(min=1, max=15, step=1, step_type="int"),
         ),
-        MessageTextInput(name="text_key", display_name="Text Key", info="Key to be used as text.", advanced=True),
+        MessageTextInput(
+            name="text_key",
+            display_name="Text Key",
+            info="Key to be used as text.This is useful cause more Parsers/ splitters look into the Text column for processing",
```

Comment:
something like: info="Key that identifies the field to be used as the text content."